### PR TITLE
First working prototype of to-do list react app 

### PR DESCRIPTION
**Pull request by amon-cofie**
#### I created the following components:
- **TodoContainer.js** : The main container for the entire todo app 
- **TodoList.js** : The container for the list of todo items on the screen
- **TodoItem.js** : The component of each todo item
- **Header.js** : The header component that displays the title of the page
- **InputTodo.js** : The form component that enables a user to enter a new todo item
- **Navbar.js** : The navbar component that allows a user to switch between different pages through routing
- **About.js** : The about page of the single page application displayed through routing
- **NotMatch.js** : The page that renders when an invalid page is being accessed

I added styling with **App.css** and **TodoItem.module.css**

### CODE REVIEW

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
Hi @amon-cofie :wave: 

Great job working on your project :1st_place_medal: 
There are just a few changes to make but you're almost there.


## Highlights :100: 

- [ ] Followed Gitflow :heavy_check_mark:
- [ ] Good commit messages :heavy_check_mark:
- [ ] Linters are passing :heavy_check_mark:
- [ ] Descriptive Pull request :heavy_check_mark:


## Required Changes :recycle: 
In the README.md file, great job giving detailed instructions on the setup and installation but you didn't include your details as the author, and the README is also missing some sections. Please refer to the [Microverse README template](https://github.com/microverseinc/readme-template) for a clear picture of how this is done. This will help make your documentation more professional.

- [ ] Please check the comments for other requested changes.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me @NahnahAJ if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/functionBased/App.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  justify-content: space-evenly;
+}
+
+.input-text {
+  font-size: 1rem;
+  font-weight: 400;
+  width: 85%;
+  padding-right: 5px;
+  padding-left: 10px;
+  border-radius: calc(0.5 * 100px);
```
[the remark this answers is not in the document]
Comment (amon-cofie) [reply src/functionBased/App.css:44]: Thank you very much for the suggestion @NahnahAJ  I will consider using that.

File: src/functionBased/components/TodoItem.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        }}
+        onKeyDown={handleUpdatedDone}
+      />
+    </li>
+  );
+};
+
+TodoItem.propTypes = {
+  todo: {
+    completed: PropTypes.bool,
```
[the remark this answers is not in the document]
Comment (amon-cofie) [reply src/functionBased/components/TodoItem.js:75]: I will implement that shortly. Thank you.

**Review by NahnahAJ — APPROVED** [review]
Hi @amon-cofie :wave: 

## Status: APPROVED :green_circle: 
Great job implementing the changes requested. :champagne: 
Your project is complete. There is nothing else to say other than its time to merge it :tada: 
Congratulations! 🎉 👏 👏 💯



Cheers and Happy coding! :clap: :clap:

**Pull request merged**